core/zero_copy_buf.rs

```diff
@@ -23,7 +23,8 @@ pub type BufVec = SmallVec<[ZeroCopyBuf; 2]>;
 pub struct ZeroCopyBuf {
   backing_store: v8::SharedRef<v8::BackingStore>,
   byte_offset: usize,
-  byte_length: usize,
+  // TODO: Determine whether it's worth exposing just to assert the size
+  pub byte_length: usize,
```

Comment:
This is supposed to be private, but I needed it to check that the size of the provided buffer is the expected one.
I don't think exposing that data is harmful. Let me know what you prefer.

If we want to keep this field private, we can also manually pass the size of the instantiated buffer as part of `Value`. It's a bit more work but it's doable.

Reply:
Please revert, you don't need to change `ZeroCopyBuf` to check length; `ZeroCopyBuf` derefs to `[u8]` and you can use its `size()` to check it.